utils/node_local_test.py

```diff
@@ -1059,24 +1059,24 @@ def get_cmd_prefix(self):
                                          suffix='.memcheck', delete=False) as log_file:
             self._xml_file = log_file.name
 
-        cmd = ['valgrind', '--fair-sched=yes']
+        cmd = ['valgrind',
+               f'--xml-file={self._xml_file}',
+               '--xml=yes',
+               '--fair-sched=yes',
+               '--gen-suppressions=all',
+               '--error-exitcode=42']
```

Comment:
This change is not required for this PR and is not a functional change however moves around the command line options to valgrind so that 'ps' can show the logfile name and therefore the running test.